index.html

```diff
@@ -42,5 +41,182 @@ <h5>LET'S CONNECT</h5>
+                    </div>
+                   
+                </div>
+                <div class="works-flex-item">
+                    <div class="btn-see">
+                        <a href="#" class="btn-see-project">See project</a>
+                    </div>
+                </div>
+            </div>
+            <div class="works-flex">
+                <div class="works-flex-item work-image">
+                    <img src="assets/images/portfolio_snapshoot/snapshoot4.png" class="img-style" alt="tonic works" />
+                </div>
+                <div class="works-flex-item">
+                    <div class="intro-text-works">Multi-Post <br/> Stories</div>
+                </div>
+                <ul class=" works-flex-item titles-container">
+                    <li class="list-main">CANOPY</li>
+                    <li class="titles-item">Back End Dev</li>
+                    <li class="titles-item">2015</li>
+                </ul>
+                <div class="works-flex-item works-description">
+                   A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <div class="works-flex-item">
+                    <div class="lang-container">
+                        <div class="lang">html</div>
+                        <div class="lang">css</div>
+                        <div class="lang">javascript</div>
+                    </div>
+                   
+                </div>
+                <div class="works-flex-item">
+                    <div class="btn-see">
+                        <a href="#" class="btn-see-project">See project</a>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="aboutme">
+            <div class="aboutme-wrapper">
+                <div class="headline-container">
+                    <div class="intro-text-about">About<br> Myself</div>
+                    <div class="intro-message-about">Hello, I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+                    <div class="connect">
+                        <h5>LET'S CONNECT</h5>
+                    </div>
+                </div>
+                
+                <div class="social">
+                    <div class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></div>
+                    <div class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></div>
+                    <div class="social-media"><a href="#"><i class="fa fa-medium"></i></a></div>
+                    <div class="social-media"><a href="#"><i class="fa fa-github"></i></a></div>
+                    <div class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></div>
+                </div>
```

Comment:
- [ ] You have implemented the social section really well according to the Figma in terms of design 👏👏. now the only thing I'd like to point out is please kindly use a `ul` tag here with `li` children. a `ul` tag signifies that you have an unordered list of data. and each social-media icon is a `list-item` (`li`). So in order to keep in line with correct HTML semantics please make this change thank you. 

something like this would be great:
```suggestion
                <ul class="social">
                    <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
                    <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
                    <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
                    <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
                    <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
                </ul>
```